### PR TITLE
Added support for seccomp

### PR DESCRIPTION
**Pull request by docbobo**
The official jessie build of docker on aarch64 will probably require ```seccomp```. This is available via ```jessie-backports``` and is installed along with docker ```17.03.0-ce```.

@StefanScherer no need to create a release right away.

### CODE REVIEW

**Comment by docbobo** [comment]
@StefanScherer any chance you could have a look?

**Comment by StefanScherer** [comment]
LGTM. So you have Docker 17.03 👍

File: versions.config [hunk 1]
```diff
@@ -8,9 +8,9 @@ RAW_IMAGE_VERSION="v0.2.2"
 RAW_IMAGE_CHECKSUM="fe9af7686960d5d2dd6c364197e196d751f6adedd7da515ed6521c31e9582f8a"
 
 # specific versions of kernel/firmware and docker tools
-# export KERNEL_VERSION="107-1"
-export DOCKER_DEB_URL="https://github.com/DieterReuter/docker-armbuilds/releases/download/v1.13.1/docker-engine_1.13.1-0.debian-jessie_arm64.deb"
-export DOCKER_DEB_CHECKSUM="dd7875ee1f62a9784c96cf22cd3dedbe61704ac5c2e054d6cfc914ac0e48de77"
+export KERNEL_VERSION="108-1"
+export DOCKER_DEB_URL="https://github.com/docbobo/docker/releases/download/v17.03.0-ce/docker-engine_17.03.0.ce-0.debian-jessie_arm64.deb"
+export DOCKER_DEB_CHECKSUM="7de1a102015122c592d196d0936e8580f7ff591ef2ad93857370f33874dea1ed"
 export DOCKER_COMPOSE_VERSION="1.9.0-23"
```
Comment (StefanScherer) [inline versions.config:14]: On the RPi I switched to use `pip install docker-compose` to get the latest version. This can be done in another PR.

**Review by StefanScherer — COMMENTED** [review]
LGTM

File: versions.config (hunk 1 above)
Comment (docbobo) [reply versions.config:14]: Yes, I know. I had that running a while ago, but I am still not sure if I really prefer this mixed mode over a clean install via apt-get. Let me check if there's a good way to pin the version. But I agree, that'll be another PR.

**Pull request merged**